build.xml

```diff
@@ -163,23 +126,6 @@
     <delete file="libraries" />
   </target>
 
-  <!-- Builds a Lightning code base from legacy Drush make files. -->
```

Comment:
True. I got rid of it because I've never used this target for anything, ever (if you have, I can restore these lines), so I decided it was just cluttering up the build file. The only thing that builds Lightning using drush make is drupal.org and our package target has worked beautifully for that.